app/controllers/interstitial_controller.rb

```diff
@@ -23,8 +23,8 @@ def redirect_param_same_as_host?
 
     param_host = begin
                    URI.parse(URI.decode(params[:redirect_to]))
-                 rescue URI::InvalidURIError
-                   URI.parse(params[:redirect_to])
+    rescue URI::InvalidURIError
+      URI.parse(params[:redirect_to])
                  end
```

Comment:
Hmm, this doesn't quite look right, did the auto-fixer do something wonky here?

Reply:
Yes this is what the autofix gave me. I've pushed up a change